.github/ISSUE_TEMPLATE.md

```diff
@@ -12,7 +13,7 @@ EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.
 ### What happened
 EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.
 
-### Expected Behavior
+### What you expected to happen
```

Comment:
Nope, it is a template that is being used by a "user" and he submits the stuff. It is not that the headers will disappear (e.g. those are not Form labels).

What you expected is okay in the moment the Form is up, but afterwards it is wrong.